demos/mqtt/mqtt_demo_basic_tls/mqtt_demo_basic_tls.c

```diff
@@ -655,9 +769,18 @@ static void eventCallback( MQTTContext_t * pMqttContext,
         switch( pPacketInfo->type )
         {
             case MQTT_PACKET_TYPE_SUBACK:
-                LogInfo( ( "Subscribed to the topic %.*s.\n\n",
-                           MQTT_EXAMPLE_TOPIC_LENGTH,
-                           MQTT_EXAMPLE_TOPIC ) );
+
+                /* Decode SUBACK and update globalSubAckStatus accordingly.  */
```

Comment:
The comment needs to be more descriptive here as well  with something like
"We have received the server response for the subscription request. The SUBACK response contains the status code response for the single subscription topic we requested subscription to. We will parse the SUBACK packet to obtain the status code, and store the status code in the global variable, #globalSubAckStatus"